### PR TITLE
Adds Stop/Start Stream button, Minor UI improvements

### PR DESCRIPTION
**Pull request by cdknorow**

### CODE REVIEW

File: webui/src/components/Layout/NavBar.js [hunk 1]
```diff
@@ -86,14 +86,16 @@ const NavBar = (props) => {
...
-                <InconSelector index={index}> </InconSelector>
-              </ListItemIcon>
-              <ListItemText primary={text} />
-            </ListItem>
-          ))}
+          {["Device Info", "Test Stream", "Configure Gateway"].map(
+            (text, index) => (
+              <ListItem button key={text} onClick={handleMenu(index)}>
+                <ListItemIcon>
+                  <InconSelector index={index}> </InconSelector>
```
Comment (jublin) [inline webui/src/components/Layout/NavBar.js:93]: Icon vs Incon

File: webui/src/components/SensorStream/StreamChart.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ const StreamChart = (data) => {
       data={data.data}
       layout={{
         autosize: true,
+        width: 1080,
```
Comment (jublin) [inline webui/src/components/SensorStream/StreamChart.js:10]: should this be a minwidth?

Comment (cdknorow) [reply webui/src/components/SensorStream/StreamChart.js:10]: plotly only takes int values :/

File: webui/src/components/Layout/NavBar.js (hunk 1 above)
Comment (cdknorow) [reply webui/src/components/Layout/NavBar.js]: fixed

**Review by jublin — APPROVED** [review]

**Pull request merged**